build_ssht_cffi.py

```diff
@@ -1,147 +1,84 @@
-import os
```

Comment:
I can't seem to make a comment on the `__init__.py` file, so I'll ask the question here: Why do we need an `__init__.py` in the top level of the repo? Do we not use the traditional package organization here?

Reply:
It's an artifact of my convoluted process of getting the restructure done (involving copying over the new package into the old repo...). I'll delete it.